upb/def.c

```diff
@@ -74,6 +74,12 @@ struct upb_fielddef {
   upb_label_t label_;
 };
 
+struct upb_extrange {
```

Comment:
I know that upb has historically biased toward very short names, but I think we should start to aim back towards the style guide.

upb_extension_range

is a lot easier to read

Reply:
Yes, I agree that the naming scheme is too terse. As discussed, we'll keep it for now for consistency, and aim towards a scheme like `upb_ExtensionRange`.